GodotAuthentication/.gitignore

```diff
@@ -1,2 +1,3 @@
 # Godot 4+ specific ignores
 .godot/
+*.meta
```

Comment:
I would make sure that all the Unity generated files (like .meta files) are deleted from the repo before merging in. Otherwise LG2M!